src/data/markdown/translated-guides/en/02 Using k6/02 Metrics.md

```diff
@@ -69,16 +69,80 @@ All the `http_req_...` lines and the ones after them are _built-in_ metrics that
+| iterations         | Counter | The aggregate number of times the VUs in the test have executed the JS script (the `default` function). |
+| iteration_duration | Trend   | The time it took to complete one full iteration. It includes the time spent in `setup` and `teardown` as well, there is currently a workaround for excluding them shown in the example below. |
 | dropped_iterations | Counter | Introduced in k6 v0.27.0, the number of iterations that could not be started due to lack of VUs (for the arrival-rate executors) or lack of time (due to expired maxDuration in the iteration-based executors). |
-| data_received      | Counter | The amount of received data. Read [this example](/examples/track-transmitted-data-per-url) to track data for an individual URL.                                                                                 |
-| data_sent          | Counter | The amount of data sent. Read [this example](/examples/track-transmitted-data-per-url) to track data for an individual URL.                                                                                     |
-| checks             | Rate    | The rate of successful checks.                                                                                                                                                                                  |
+| data_received      | Counter | The amount of received data. Read [this example](/examples/track-transmitted-data-per-url) to track data for an individual URL. |
+| data_sent          | Counter | The amount of data sent. Read [this example](/examples/track-transmitted-data-per-url) to track data for an individual URL. |
+| checks             | Rate    | The rate of successful checks. |
+
+<Collapsible title="Workaround to isolate `iteration_duration` metric from Setup and Teardown">
+
+A common requested case is to track the `iteration_duration` metric without including time spent for `setup` and `teardown` functions.
+This feature is not yet available but a threshold on `iteration_duration` can be used as a workaround. Check the following example.
+
+<CodeGroup lineNumbers={[true]}>
+
+```javascript
+import { sleep } from 'k6'
+import http from 'k6/http'
+
+export const options = {
+    vus: 1,
+    iterations: 1,
+    thresholds: {
+        // We're using 'scenario:default' because that's the internal k6
+        // scenario name when we haven't actually specified `options.scenarios`
+        // explicitly and are just using the old execution shortcuts instead...
+        'iteration_duration{scenario:default}': [
+            // Some dummy threshold that's always going to pass.
+	    // You don't even need to have something here.
+            `max>=0`,
+        ],
+        // You can isolate setup and teardown metrics, if you are interested in them
+        // for some reason. This looks nasty, but '::' is the group separator.
```

Comment:
```suggestion
        // You can remove the time spent only on `setup` or only on `teardown` function too.
        // This looks nasty, but '::' is the group separator.
```

\+ a link to [group docs](https://k6.io/docs/using-k6/tags-and-groups/#groups) here?

Reply:
this doesn't *remove* the time spent ... it makes it so the thresholds is calculated only on the provided values, so if anything it "removes" all other values for this metrics :shrug: I od think "isolate" is good enough, but we can have a better explanation *in one place*